README.md

```diff
@@ -1,32 +1,169 @@
 # Prompt flow
```

Comment:
maybe put banner infront of the title?
like what pandas does: 
![image](https://github.com/microsoft/promptflow/assets/47586720/aed6539e-2a0b-4034-a372-a5915b0e5118)